microsoft/testsuites/core/provisioning.py

```diff
@@ -112,6 +112,7 @@ def verify_deployment_provision_standard_ssd_disk(
         priority=1,
         requirement=simple_requirement(
             environment_status=EnvironmentStatus.Deployed,
+            min_core_count=8,
```

Comment:
The error message is not related to core count? What's the root cause?

Reply:
related with os disk and temp storage of using vm size

Reply:
Please check if there is another information about the storage size. The core count is not accurate to control the storage size, and it's misleading.

Reply:
we can find the info from vmsize, but need figure out how to get the os disk size.